### ISSUE TITLE
Debian Buster docker containers exiting with 132 on Ubuntu

### ISSUE THREAD

**Issue by Samira-El** [debuerreotype/docker-debian-artifacts#112]
I'm facing the same issue as https://github.com/debuerreotype/docker-debian-artifacts/issues/81

Running `docker run -it debian:buster bash` / `docker run -it debian:buster-slim bash` fails with `Existed (132)` and no logs.

Kernel logs show `traps: bash[20302] trap invalid opcode ip:7f9195e67190 sp:7fffe3429578 error:0 in libc-2.28.so[7f9195d33000+148000]`

I'm running a `ubuntu-bionic 18.04.1` OS with kernel `5.3.0-53-generic`

Docker on my machine is
``` 
Client:
 Context:    default
 Debug Mode: false
 Plugins:
  app: Docker App (Docker Inc., v0.9.1-beta3)
  buildx: Build with BuildKit (Docker Inc., v0.5.1-docker)

Server:
 Containers: 8
  Running: 0
  Paused: 0
  Stopped: 8
 Images: 3
 Server Version: 20.10.2
 Storage Driver: overlay2
  Backing Filesystem: extfs
  Supports d_type: true
  Native Overlay Diff: true
 Logging Driver: json-file
 Cgroup Driver: cgroupfs
 Cgroup Version: 1
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: inactive
 Runtimes: io.containerd.runc.v2 io.containerd.runtime.v1.linux runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: 269548fa27e0089a8b8278fc4fc781d7f65a939b
 runc version: ff819c7e9184c13b7c2607fe6c30ae19403a7aff
 init version: de40ad0
 Security Options:
  seccomp
   Profile: default
 Kernel Version: 5.3.0-53-generic
 Operating System: Ubuntu 18.04.5 LTS
 OSType: linux
 Architecture: x86_64
 CPUs: 8
 Total Memory: 31.13GiB
 Name: T480-pf1n4d3y
 ID: 6OYR:HLB4:SQ6E:B55Q:T3WS:VVYB:3CDQ:3W6R:7OWJ:R66F:XNDN:AWTF
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 Registry: https://index.docker.io/v1/
 Labels:
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false

WARNING: No swap limit support
WARNING: No blkio weight support
WARNING: No blkio weight_device support
```

CPU flags are:
```
vendor_id	: GenuineIntel
cpu family	: 6
model		: 142
model name	: Intel(R) Core(TM) i5-8250U CPU @ 1.60GHz
stepping	: 10
microcode	: 0xe0
cpu MHz		: 800.060
cache size	: 6144 KB
physical id	: 0
siblings	: 8
core id		: 0
cpu cores	: 4
apicid		: 0
initial apicid	: 0
fpu		: yes
fpu_exception	: yes
cpuid level	: 22
wp		: yes
flags		: fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush dts acpi mmx fxsr sse sse2 ss ht tm pbe syscall nx pdpe1gb rdtscp lm constant_tsc art arch_perfmon pebs bts rep_good nopl xtopology nonstop_tsc cpuid aperfmperf pni pclmulqdq dtes64 monitor ds_cpl vmx est tm2 ssse3 sdbg fma cx16 xtpr pdcm pcid sse4_1 sse4_2 x2apic movbe popcnt tsc_deadline_timer aes xsave avx f16c rdrand lahf_lm abm 3dnowprefetch cpuid_fault epb invpcid_single pti ssbd ibrs ibpb stibp tpr_shadow vnmi flexpriority ept vpid ept_ad fsgsbase tsc_adjust bmi1 avx2 smep bmi2 erms invpcid mpx rdseed adx smap clflushopt intel_pt xsaveopt xsavec xgetbv1 xsaves dtherm ida arat pln pts hwp hwp_notify hwp_act_window hwp_epp md_clear flush_l1d
bugs		: cpu_meltdown spectre_v1 spectre_v2 spec_store_bypass l1tf mds swapgs itlb_multihit
bogomips	: 3600.00
clflush size	: 64
cache_alignment	: 64
address sizes	: 39 bits physical, 48 bits virtual
power management:
```

Can someone help/guide me please?

**Comment by tianon** [comment]
There's more active discussion on #86 which is a similar issue.

**Comment by Samira-El** [comment]
Thank you Tianon, I'll keep an eye on that issue.

**Comment by tianon** [comment]
As noted in #86, are you also using Cylance?

**Comment by Samira-El** [comment]
Oh haven't seen that issue, I'm using Cylance indeed, stopping its service used to allow me to spin up the debian buster container, but it's no longer the case.

**Comment by stappersg** [comment]
Please elaborate _but it's no longer the case_.

**Comment by Samira-El** [comment]
Sure, so upon investigating this same issue about 6 months ago, I figured that Cylance was indeed the culprit, so the only option that I had to run containers was to stop cylance service. It worked for some time. Then IT department reinstalled Cylance from scratch on my machine, and it was no longer interfering with debian busters containers thus I was able to run them while Cylance was running. However, lately, it's no longer consistent, whether I stop cylance or not, I cant spin up containers, sometimes, stopping Cylance does allow me to though.

Cylance logs don't show anything about anything docker related.
